crates/core/src/worker_metrics/mod.rs

```diff
@@ -257,6 +257,11 @@ metrics_group!(
         #[help = "The cumulative number of bytes sent to clients"]
         #[labels(txn_type: WorkloadType, db: Identity)]
         pub bytes_sent_to_clients: IntCounterVec,
+
+        #[name = spacetime_client_connection_incoming_queue_length]
+        #[help = "The number of client -> server WebSocket messages waiting in a client connection's incoming queue"]
+        #[labels(db: Identity, client_identity: Identity, connection_id: ConnectionId)]
```

Comment:
I think we should avoid adding any metrics with per-client or per-connection labels. I'm not sure how big the impact would be on our metrics collection/storage/querying performance, but the cardinality of a stat like this can blow up very quickly.

Reply:
As discussed out-of-band, I'll alter this to be a per-database metric, which I suppose will be the sum of the length of all clients' queues. I expect to open a separate PR for that, since we want to use this PR in testing.